packages/sync/src/links/OfflineQueueLink.ts

```diff
@@ -135,11 +135,14 @@ export class OfflineQueueLink extends ApolloLink {
   public openQueueOnNetworkStateUpdates(): void {
     const self = this;
     if (this.networkStatus) {
-      if (this.networkStatus.isOffline()) {
-        this.close();
-      } else {
-        this.open();
-      }
+      this.networkStatus.isOffline().then(offline => {
```

Comment:
I believe that we need to set initial state before we restore queue.
Happy to do that in other PR.